src/SPC/builder/unix/library/librdkafka.php

```diff
@@ -26,42 +26,18 @@ public function patchBeforeBuild(): bool
 
     protected function build(): void
     {
-        UnixAutoconfExecutor::create($this)
-            ->appendEnv(['CFLAGS' => '-Wno-int-conversion -Wno-unused-but-set-variable -Wno-unused-variable'])
```

Comment:
please test with zig and clang on linux, both failed witho -Wno-int-conversion last time

Reply:
Just tested with `native-native-musl` and `native-native-gnu.2.17` target. It works well.